File: leetcode/interview/1.two-sum.js
/*
 * @lc app=leetcode id=1 lang=javascript
 *
 * [1] Two Sum
 */

// @lc code=start
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for(i=0;i<nums.length;i++){
        var correctValue=target-nums[i];
        for(j=i+1;j<nums.length;j++){
            if(nums[j]==correctValue){
                return [i,j];
            }
        }
    }
};
// @lc code=end

File: leetcode/c#-interview/1.two-sum.cs
/*
 * @lc app=leetcode id=1 lang=csharp
 *
 * [1] Two Sum
 */

// @lc code=start
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
         var total=0;
    for(int i=1;i<nums.Length;i++){
        for(int j=0;j<i;j++){
            if(nums[j]+nums[i]==target){
                return new int[]{j,i};
            }
        }
    }
        return null;
    }
}
// @lc code=end

